demo/index.html

```diff
@@ -912,7 +912,7 @@ <h3>Custom Drag Image</h3>
 
         // tmp fix for iOS10 touchmove bug (https://github.com/timruffles/ios-html5-drag-drop-shim/issues/77)
         window.addEventListener('touchmove', function () {
-        })
+        }, {passive: false})
```

Comment:
Did you have a chance to test what happens on browsers that don't support the new `addEventListener` API? Does it still work on iOS 10 this way?